src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -64,11 +63,28 @@ type FlowNode = TFlowNode | TActivity | TReceiveTask | TEventBasedGateway | TTex
  */
 export default class ProcessConverter {
   private defaultSequenceFlowIds: string[] = [];
+  private elementsWithoutParentByProcessId: Map<string, ShapeBpmnElement[]> = new Map();
 
   constructor(private convertedElements: ConvertedElements, private parsingMessageCollector: ParsingMessageCollector) {}
 
   deserialize(processes: string | TProcess | (string | TProcess)[]): void {
     ensureIsArray(processes).forEach(process => this.parseProcess(process));
+    // Need to call this after all processes have been parsed
+    ensureIsArray(processes).forEach(process => this.assignParentOfProcessElementsCalledByCallActivity(process));
+  }
+
+  private assignParentOfProcessElementsCalledByCallActivity(process: TProcess): void {
```

Comment:
Here, we only use the `id` of the `process` parameter, so we should pass it directly to the method. This will make the signature more explicit